frontend/solidus_frontend.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'solidus_api', s.version
   s.add_dependency 'solidus_core', s.version
 
-  s.add_dependency 'canonical-rails', '~> 0.2.0'
+  s.add_dependency 'canonical-rails', '~> 0.2.10'
```

Comment:
we can probably revert this gemspec change now?

Reply:
This PR moves from the deprecated `whitelisted_params` to the new `allowed_params`, which works in 0.2.10 and 0.2.11, so I think that we are going to want something to lock the version at 0.2.10 or 0.2.11 in order to ensure compatibility. The `allowed_params` method is not compatible with 0.2.9 and lower.

Reply:
Oh, you are totally right.